monai/bundle/scripts.py

```diff
@@ -541,11 +541,15 @@ def load(
     return model
 
 
+@deprecated_arg_default("tag", "hosting_storage_v1", "dev", since="1.3", replaced="1.5")
```

Comment:
we can set it to `since="1.2"` to start using the new logic now, `since="1.3"` means using it after tagging v1.3

```suggestion
@deprecated_arg_default("tag", "hosting_storage_v1", "dev", since="1.2", replaced="1.5")
```